### PR TITLE
pangea-sdk: add Sanitize docs (GEA-13292)

### PR DESCRIPTION
**Pull request by kenany**

### CODE REVIEW

File: packages/pangea-sdk/pangea/services/sanitize.py [hunk 1]
```diff
@@ -139,6 +218,44 @@ def request_upload_url(
...
+            size: The size (in bytes) of the file. If the upload doesn't match, the call will fail.
+            crc32c: The CRC32C hash of the file data, which will be verified by the server if provided.
+            sha256: The hexadecimal-encoded SHA256 hash of the file data, which will be verified by the server if provided.
+            uploaded_file_name: Name of the user-uploaded file, required for `TransferMethod.PUT_URL` and `TransferMethod.POST_URL`.
+
+        Raises:
+            PangeaAPIException: If an API error happens.
+
+        Returns:
+            The sanitized file and information on the sanitization that was
```
Comment (pangea-andrest) [inline packages/pangea-sdk/pangea/services/sanitize.py:244]: In `request_upload_url` we do not return a sanitized file or its information, we should just return presigned url info, then, after upload the file, we should poll the result

**Review by pangea-andrest — CHANGES_REQUESTED** [review]

File: packages/pangea-sdk/pangea/services/sanitize.py (hunk 1 above)
Comment (kenany) [reply packages/pangea-sdk/pangea/services/sanitize.py]: Corrected.

**Review by pangea-andrest — APPROVED** [review]

File: packages/pangea-sdk/pangea/services/sanitize.py [hunk 2]
```diff
@@ -95,6 +135,44 @@ def sanitize(
         uploaded_file_name: Optional[str] = None,
         sync_call: bool = True,
     ) -> PangeaResponse[SanitizeResult]:
+        """
+        Sanitize
+
+        Apply file sanitization actions according to specified rules. Beta API.
```
Comment (pangea-andrest) [inline packages/pangea-sdk/pangea/services/sanitize.py:141]: According to yesterday talk, should we bold `Beta API` or move next to the "title" `Sanitize`?

File: packages/pangea-sdk/pangea/asyncio/services/sanitize.py [hunk 3]
```diff
@@ -4,11 +4,10 @@
...
 
 import pangea.services.sanitize as m
+from pangea.asyncio.services.base import ServiceBaseAsync
 from pangea.response import PangeaResponse, TransferMethod
 from pangea.utils import FileUploadParams, get_file_upload_params
 
-from .base import ServiceBaseAsync
-
 
 class SanitizeAsync(ServiceBaseAsync):
```
Comment (pangea-andrest) [inline packages/pangea-sdk/pangea/asyncio/services/sanitize.py:12]: Are we missing docs here?

**Review by pangea-andrest — APPROVED** [review]

File: packages/pangea-sdk/pangea/services/sanitize.py (hunk 2 above)
Comment (kenany) [reply packages/pangea-sdk/pangea/services/sanitize.py:141]: I'm gonna tackle all of them (Share + Sanitize) at once after this.

**Pull request merged**